modules/signals/entities/src/index.ts

```diff
@@ -1,5 +1,7 @@
 export { addEntity } from './updaters/add-entity';
+export { prependEntity } from './updaters/prepend-entity';
 export { addEntities } from './updaters/add-entities';
+export { prependEntities } from './updaters/prepend-entities';
```

Comment:
exports should be sorted alphabetically:

```suggestion
export { addEntities } from './updaters/add-entities';
export { prependEntity } from './updaters/prepend-entity';
export { prependEntities } from './updaters/prepend-entities';
```

Reply:
Great suggestion — I've removed the redundant check and replaced push + reverse with unshift as proposed. Thanks for the optimization tip!

Reply:
fixed, thanks